packages/code-du-travail-ui/src/ToggleButton/index.js

```diff
@@ -4,24 +4,34 @@ import styled, { css } from "styled-components";
 import { animations, box, colors, fonts, spacing } from "../theme";
 import { darken, lighten, transparentize } from "polished";
 
-// OK it is a simple button in fact, on which you can provid a pressed prop...
-const ToggleButton = ({ pressed, onClick, ...props }) => {
-  return <Button aria-pressed={pressed} {...props} onClick={onClick} />;
+// We don't want the kind prop to be passed down to the button
+// eslint-disable-next-line
+const Button = ({ pressed, kind, onClick, ...props }) => {
+  return <button aria-pressed={pressed} {...props} onClick={onClick} />;
 };
 
-ToggleButton.propTypes = {
-  pressed: PropTypes.bool,
-  onClick: PropTypes.func
+Button.propTypes = {
+  kind: PropTypes.oneOf([
```

Comment:
I would prefer to use  `variant` for that prop name  with default on "primary"
also why not use flavors ?

Reply:
Agree with variant. With the default on primary however, this would cause regressions. The absence of "variant / kind" is also a style in itself (cf. story book)

Reply:
We could change that in another PR

Reply:
ok, can we also have a variant="clear" or something like which is like the default ?

Reply:
Hum we could have something like `variant="default"` ?